Libraries/libcar/Sources/Rendition.cpp

```diff
@@ -227,7 +188,9 @@ Load(
                         };
                         slices.push_back(slice);
                 }
-                rendition.slices() = std::move(slices);
+                if (slices.size() > 0) {
```

Comment:
Ah. There should be a default slice that is the whole image. Right now that's a special case in the write method if the size is zero.